tests/Fixer/FunctionNotation/MethodArgumentSpaceFixerTest.php

```diff
@@ -983,47 +1003,4 @@ functionCall(
 
         $this->doTest($expected, $input);
     }
-
-    /**
-     * @param string      $expected
-     * @param null|string $input
-     * @param null|array  $config
-     *
-     * @requires PHP 7.3
-     * @dataProvider provideFix73Cases
-     */
-    public function testFix73($expected, $input = null, array $config = null)
```

Comment:
ping @SpacePossum ,
next time `array $config = []` like everywhere else, please ;)

Reply:
:+1:  sorry about that!

Reply:
dont worry, no big deal ;)